pallets/communities/README.md

```diff
@@ -0,0 +1,179 @@
+# Communities Pallet
+
+Part of the People Local Interactions Protocol, this pallet enables people
+to unite and create local communities that share a common interest or
+economic activity. In simpler terms, it can be considered a DAO Factory.
+
+## Overview
+
+The Communities pallet provides functionality for managing communities,
+facilitating its participants to have governance over the community entity
+(and its associated account), and running economic activities:
+
+- Community registration and removal.
+- Validating a community challenge.
+- Handling a community-governed treasury account.
+- Enrolling/removing members from a community.
+- Promoting/demoting members within the community.
+- Running proposals to enable community governance.
+- Issue governance tokens.
+- Issue economic (sufficient) tokens.
+
+## Terminology
+
+- **Community:** An entity comprised of _members_ —each one defined by their
+  [`AccountId`][1]— with a given _description_ who can vote on _proposals_
+  and actively take decisions on behalf of it. Communities are given a
+  _treasury account_ and can issue tokens. It is required that a community
+  contributes to the network to be active and operate within it.
+- **Community Description:** A set of metadata used to identify a community
+  distinctively. Typically, a name, a list of locations (given as a list of
+  one or more [`H3Index`][2]), and a list of URL links.
+- **Community Status:** A community can be either `awaiting`, `active`, or
+  `frozen` depending on whether the community has proven via a challenge
+  it's actively contributing to the network with infrastructure provisioning
+  (i.e. a [collator][3] node) or by depositing funds.
+- **Validity Challenge:** A proof that a community is actively contributing
+  to the network. The mechanisms for challenge verification are usually
+  checked via an off-chain worker. Still, it's possible for a trusted origin
+  to manually mark a community challenge as passed, effectively changing the
+  status of the community to `active`.
+- **Admin:** An [`AccountId`][1] registered into the community that is set
+  as such. Can call [privileged functions](#privileged-functions) within the
+  community.
+- **Member:** An [`AccountId`][1] registered into the community as such. Can
+  have a rank within it and vote in the community's polls.
+- **Member Rank:** Members could have a rank within the community. This can
+  determine a voting weight depending on the community's voting mechanism.
+- **Proposal:** A poll with an optionally set duration that executes a
+  [call][4] dispatch if approved when it's closed.
+- **Treasury Account:** A keyless [`AccountId`][1] generated on behalf of
+  the community. Can receive [payments][5], transfers, or payment [fees][6].
+  It can transfer funds via a privileged call executed by the community
+  _admin_ or a call dispatched from a proposal.
+- **Governance Token:** A [non-sufficient fungible asset][7] issued and
```

Comment:
@olanod @pandres95 why not sufficient fungible asset? Non sufficient assets are quite a pain from the UX perspective.